sites/docker-compose.berwyn.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+
+services:
+    hub-be:
+        environment:
+            DRUPAL_APP_URI: https://drupal.digital-hub-stage.hmpps.dsd.io
+            HUB_DB_ENV_MYSQL_DATABASE: hubdb
+            HUB_DB_ENV_MYSQL_USER: hubdb_user_2y478gt
+            HUB_DB_PORT_3306_TCP_ADDR: hub-db
+            HUB_EXT_FILE_URL: http://digital-hub.bwi.dpn.gov.uk:11001/sites/default/files
+            PHP_MEMORY_LIMIT: 256M
+            PHP_UPLOAD_MAX_FILE_SIZE: 1024M
+            PHP_POST_MAX_SIZE: 1024M
```

Comment:
Should we reduce the UPLOAD and POST max sizes to 500M to match staging?  Only staging has any content uploads done.  While not really an issue, I feel warm and fuzzy with the consistency.